src/main/java/org/idea/plugin/atg/inspection/DuplicatedPropertyInspection.java

```diff
@@ -0,0 +1,84 @@
+package org.idea.plugin.atg.inspection;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.lang.properties.IProperty;
+import com.intellij.lang.properties.psi.impl.PropertiesFileImpl;
+import com.intellij.lang.properties.psi.impl.PropertyImpl;
+import com.intellij.openapi.components.ServiceManager;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.TextRange;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementVisitor;
+import org.idea.plugin.atg.AtgToolkitBundle;
+import org.idea.plugin.atg.index.AtgIndexService;
+import org.idea.plugin.atg.util.AtgComponentUtil;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class DuplicatedPropertyInspection extends LocalInspectionTool {
```

Comment:
1. Name confuses, as identical inspection is present already and checks of presence same key in given file. Probably PropertyOverridedWithSameValue.. or some like that corresponds better
2. In current implementation it will show warning in both Parent/Child module, but warning should be valid for child ones, and ignored on independent modules (from my point of view)